DXRVanilla/DeusEx/Classes/PersonaScreenInventory.uc

```diff
@@ -129,3 +134,147 @@ event DescendantRemoved(Window descendant)
     }
     Super.DescendantRemoved(descendant);
 }
+
+function CreateButtons()
```

Comment:
Why not just call Super.CreateButtons and then add the btnRefusal after?

Reply:
Because Super.CreateButtons adds the "Change Ammo" button, which is being removed because it's dumb and taking up space.

Reply:
`Super.CreateButtons` doesn't provide access to the `PersonaButtonBarWindow` it creates, which is necessary to add the button.

Plus the new code needs to be in the middle of the function, not before or after it. Running it before the `PersonaButtonBarWindow` is created (if it hypothetically had access to it `winActionButtons`) would do nothing and running after would put the button at the far left.